test-app/src/widgets/ComboBox.js

```diff
@@ -27,6 +27,14 @@ export default class ComboBox extends createjs.Container {
     this._dropDownView.visible = false;
   }
 
+  setTabbable(tabbable) {
```

Comment:
Added so the tabindex can be flipped when the combobox shouldn't be in the tab order; in our case this is when the dialog is open in the Form example and the combobox is behind it